src/main/java/com/devcycle/sdk/server/local/DVCLocalClient.java

```diff
@@ -1,25 +1,47 @@
 package com.devcycle.sdk.server.local;
 
+import com.devcycle.sdk.server.common.api.DVCApi;
+import com.devcycle.sdk.server.common.api.DVCApiClient;
 import com.devcycle.sdk.server.common.model.*;
-
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import retrofit2.Call;
+import retrofit2.Response;
+import java.io.IOException;
 import java.util.Collections;
+import java.util.Map;
 import java.util.Objects;
 
-public final class DVCClient {
+
+public final class DVCLocalClient {
+
+  private final DVCApi api;
+  private final DVCOptions dvcOptions;
+
+  private static final LocalBucketing localBucketing = new LocalBucketing();
```

Comment:
I don't think we want this to be final; static alone is fine.

Reply:
fixed!